#! /bin/sh /usr/share/dpatch/dpatch-run
## 91-patch-dg-basedir.dpatch by Thomas Girard <thomas.g.girard@free.fr>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix path to DependencyGenerator.

@DPATCH@
--- ACE_wrappers.orig/bin/depgen.pl	2008-12-09 05:42:49.000000000 +0100
+++ ACE_wrappers/bin/depgen.pl	2010-12-22 15:15:11.046305586 +0100
@@ -19,12 +19,8 @@
 use File::Spec;
 use File::Basename;
 
-my($basePath) = $FindBin::RealBin;
-if ($^O eq 'VMS') {
-  $basePath = File::Spec->rel2abs(dirname($0)) if ($basePath eq '');
-  $basePath = VMS::Filespec::unixify($basePath);
-}
-unshift(@INC, $basePath . '/DependencyGenerator');
+my($basePath) = '/usr/lib/ace/bin/DependencyGenerator';
+unshift(@INC, $basePath);
 
 my($mpcroot) = $ENV{MPC_ROOT};
 my($mpcpath) = (defined $mpcroot ? $mpcroot :
